### PR TITLE
bugfix/COR-1772_legend_threshold_values

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard30**
- Changed threshold values for sewage water and patients choropleths
 - Created switch cases for pagetypes
 - Added new key to parse second value in threshold
 - Deleted unused key

### CODE REVIEW

File: packages/app/src/components/choropleth/logic/thresholds.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ const sewerThresholds: ChoroplethThresholdsValue[] = [
   },
   {
     color: colors.scale.blueDetailed[0],
-    threshold: 0,
+    threshold: 0.001,
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/components/choropleth/logic/thresholds.ts:100]: Please add a small comment about why you are doing this.

**Review by VWSCoronaDashboard29 — APPROVED** [review]

File: packages/app/src/components/choropleth-legenda.tsx [hunk 2]
```diff
@@ -29,17 +29,28 @@ export function ChoroplethLegenda({ title, thresholds, valueAnnotation, pageType
...
+      case 'patienten-in-beeld' || 'ziekenhuis-opnames':
+        if (i === 0 && x.threshold === 0) {
+          label = commonTexts.common.no_notifications;
+        } else if (i === 1) {
+          label = replaceVariablesInText(commonTexts.common.bigger_than_zero_and_less_than_value, {
+            value_1: formatNumber(thresholds[i + 1].threshold),
+          });
+        }
+        break;
+      default:
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/components/choropleth-legenda.tsx:52]: The default clause is empty, indicating that it is not necessary.

**Review by VWSCoronaDashboard28 — COMMENTED** [review]
Please see my comments

File: packages/app/src/components/choropleth/logic/thresholds.ts (hunk 1 above)
Comment (VWSCoronaDashboard30) [reply packages/app/src/components/choropleth/logic/thresholds.ts:104]: done!

File: packages/app/src/components/choropleth-legenda.tsx (hunk 2 above)
Comment (VWSCoronaDashboard30) [reply packages/app/src/components/choropleth-legenda.tsx]: You're right! I thought it's always needed. My bad!

**Review by VWSCoronaDashboard29 — APPROVED** [review]

**Pull request merged**